### DISCUSSION TITLE
search tracks by metadata

### DISCUSSION THREAD

**Discussion by fokxon** [GMOD/jbrowse-components#2292] [Q&A] [answered]
In jbrowse 1, we can search for tracks based on the metadata using faceted track selector. Is there a way to achieve the same thing in jbrowse 2?

**Comment by cmdcolin** [comment, marked as answer]
Hi there, this is not done currently but could be added. Currently the search box matches on both the category name and trackname . I also have a request for searching metadata here https://github.com/GMOD/jbrowse-components/issues/1708

**Reply by cmdcolin** [reply]
Note that there is also a separate request for adding the "faceted" track selector too
